app/models/performance_dashboards/overview.rb

```diff
@@ -57,80 +57,13 @@ def self.available_chart_types
 
   protected def build_control_sections
     [
-      build_general_control_section,
+      build_general_control_section(include_inactivity_days: true),
       build_coc_control_section,
       build_household_control_section,
       build_demographics_control_section,
     ]
   end
 
-  def exiting_by_destination
```

Comment:
Was this just dead code?

Reply:
Thanks!  yes, it looks like they were moved here: `app/models/performance_dashboards/household.rb` Though maybe not used even from there.

Reply:
Yeah I was noticing that the `household` report has been disabled for over a year, maybe that report is dead code that can be removed as well?

Reply:
Correct!  We removed the household report in favor of a Head of Households only checkbox, which accomplished the same thing.  We can remove the Household code in a separate PR.